client/angular/controllers/MainController.js

```diff
@@ -51,9 +51,19 @@
         };
 
         // Ordering function such that newer tweets precede older tweets
+        // Pinned tweets are always placed before unpinned tweets
         var chronologicalOrdering = function(tweetA, tweetB) {
-            return new Date(tweetB.created_at).getTime() - new Date(tweetA.created_at).getTime();
```

Comment:
Nah I thought that too but overflowing pinned tweets will be added to the start of the next column, preserving their order without affecting the order of the existing tweets in that column, so it's all good. The magic of the pipeline!